src/index.html

```diff
@@ -8,9 +8,41 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+    method="POST"
+    class="form form--big-search-form"
+    data-qa="big"
+    >
```

Comment:
1. apply code styling recommendations from a checklist all over the page
2. you can simplify modifiers names a little
```suggestion
    <form 
      action="#"
      method="POST"
      class="form form--big"
      data-qa="big"
    >
```